internal/controller/postgrescluster/instance_test.go

```diff
@@ -1573,3 +1576,178 @@ func TestFindAvailableInstanceNames(t *testing.T) {
 		})
 	}
 }
+
+func TestReconcileInstanceSetPodDisruptionBudget(t *testing.T) {
+	env, cc, config := setupTestEnv(t, ControllerName)
+	t.Cleanup(func() { teardownTestEnv(t, env) })
+
+	r := &Reconciler{}
+	ctx, cancel := setupManager(t, config, func(mgr manager.Manager) {
```

Comment:
🔧 This test is calling reconcile methods directly, so there should be no Manager.